PICMI_Python/fields.py

```diff
@@ -14,7 +14,16 @@ class PICMI_ElectromagneticSolver(_ClassWithInit):
     """
     Electromagnetic field solver
       - grid: grid object to be used by the solver (grid object)
-      - method: One of 'Yee', 'CKC', 'Lehe', 'PSTD', 'PSATD', 'GPSTD' or 'ECT' (string)
+      - method: One of 'Yee', 'CKC', 'Lehe', 'PSTD', 'PSATD', 'GPSTD', 'DS', or 'ECT' (string)
+        - 'Yee': standard solver using the staggered Yee grid (https://doi.org/10.1109/TAP.1966.1138693)
+        - 'CKC': solver with the extended Cole-Karkkainen stencil with better dispersion properties
+          (https://doi.org/10.1103/PhysRevSTAB.16.041303)
```

Comment:
```suggestion
          (https://doi.org/10.1103/PhysRevSTAB.16.041303 https://doi.org/10.1103/PhysRevSTAB.16.041303)
```

Reply:
What was changed? It's the same doi reference.

Reply:
yes, that's the same. I wanted to comment on the other line only, forgot to remove